client/src/utils/capitalizeFirstLetter.js

```diff
@@ -0,0 +1,9 @@
+export const capitalizeList = (list) => (
```

Comment:
is this for backend or for UI? for web UI this should be done in just CSS. However I know in React-native they dont but this aint react-native, yet.

Reply:
if this is just for web and just for the UI then lets keep it as CSS `text-decoration: 'capitolize'`  or something like that